opentech/apply/users/migrations/0014_add_sentinel_user.py

```diff
@@ -0,0 +1,38 @@
+# Generated by Django 2.0.13 on 2019-09-03 15:15
+
+from django.db import migrations
+
+
+SENTINEL_EMAIL = 'sentinel'
+
+
+def add_sentinel_user(apps, schema_editor):
+    User = apps.get_model('users', 'User')
+    User.objects.create(
+        email=SENTINEL_EMAIL,
+        full_name='sentinel',
+        slack='sentinel',
```

Comment:
is slack required?

Reply:
It's not nullable so I could set it to a blank string, but this seemed more obvious if someone found it exposed somewhere.  Which do you prefer, `'sentinel'` or `''`?